tabs/tab_corr.py

```
@@ -263,38 +250,38 @@ def _update_numeric_cols():
                 icc_vars = _auto_detect_icc_vars(cols)
-                
+
                 # Update selectize with all numeric cols
                 ui.update_selectize(
                     f"{namespace}_icc_vars",
                     choices=cols,
                     selected=icc_vars  # Pre-select auto-detected ones
                 )
-                
+
                 logger.info(f"Auto-detected ICC/Rater variables: {icc_vars}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)